server/opts.go

```diff
@@ -1680,6 +1680,53 @@ func parseJetStreamForAccount(v interface{}, acc *Account, errors *[]error, warn
 	return nil
 }
 
+func getInterfaceValue(v interface{}) (int64, error) {
+	_, ok := v.(int64)
+	if ok {
+		return v.(int64), nil
+	}
+
+	_, ok = v.(string)
+	if !ok {
+		return 0, fmt.Errorf("must be int64 or string")
+	}
+
+	num, err := stringStorageSize(v.(string))
+	if err != nil {
+		return 0, err
+	}
+
+	return int64(num), nil
+}
+
+func stringStorageSize(s string) (int, error) {
+	suffix := strings.ToUpper(s[len(s)-1:])
```

Comment:
This would panic if string `s` is empty. You can reproduce with:
```
jetstream {
  max_file_store: ""
}
```
You may want to add to the test.

Reply:
yeah good catch